script/tool/test/test_command_test.dart

```diff
@@ -49,16 +49,14 @@ void main() {
               'flutter', const <String>['test', '--color'], plugin2Dir.path),
         ]),
       );
-
-      cleanupPackages();
```

Comment:
Why don't we need cleanupPackages anymore?  The function isn't showing up in the diff, are we still using it somewhere else, should we remove it?

Reply:
It's removed; see script/tool/test/util.dart diff.